### PR TITLE
Add the case of srv-list for virt-admin

### PR DESCRIPTION
**Pull request by iccaszhulili**
Signed-off-by: Lily Zhu <lizhu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import re
+from autotest.client.shared import error
```
Comment (balamuruhans) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: exception from avocado.utils can be used

File: libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    """
+    Test command: virt-admin srv-list.
+    """
+    server_name = params.get("server_name", "")
+    vp = virt_admin.VirtadminPersistent()
+    result = vp.srv_list(ignore_status=True, debug=True)
+    output = result.stdout.strip()
+
+    if result.exit_status:
+        raise error.TestFail("This operation should success but failed!")
```
Comment (will-Do) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: Same with https://github.com/autotest/tp-libvirt/pull/987

File: libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py (hunk 1 above)
Comment (will-Do) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: Yes, we are encourage pople to use avocado instead of autotest

Comment (iccaszhulili) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: @balamuruhans @will-Do Thanks very much. This issue and the next one have been fixed according to your reviews.

File: libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import re
+from avocado.core import exceptions
+from virttest import virt_admin
+
+
+def run(test, params, env):
+    """
+    Test command: virt-admin srv-list.
+    """
```
Comment (will-Do) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: That will be great if you can add more desc for the test scenario

Comment (iccaszhulili) [inline libvirt/tests/src/virt_admin/monitor/virt_admin_srv_list.py]: @will-Do updated. Thx :)

**Pull request merged**